docs/architecture/adr-063-privval-grpc.md

```diff
@@ -0,0 +1,75 @@
+# ADR 063: Privval gRPC
+
+## Changelog
+
+- 23/11/2020: Initial Version (@marbar3778)
+
+## Context
+
+Validators use remote signers to help secure their keys. This system is Tendermint's recommended way to secure validators, but the path to integration with Tendermint's private validator client is plagued with custom protocols. 
+
+Tendermint uses its own custom secure connection protocol (`SecretConnection`) and a raw tcp/unix socket connection protocol. The secure connection protocol until recently was exposed to man in the middle attacks and limits the amount and speed of integrations users can make. The raw tcp connection protocol is less custom, but has been causing minuet issues with users. 
```

Comment:
minuet -> minute

unless of course you actually mean [Minuet](https://en.wikipedia.org/wiki/Minuet) 😄 

Alos, what do you mean by:

> limits the amount and speed of integrations users

Reply:
It's Tendermint's second dance type after the polka 😆

Reply:
> Alos, what do you mean by:
> 
> > limits the amount and speed of integrations users

With the current protocol if integrators would like to a language other than go they will have to rewrite the secret connection and recreate how the protocol works. Migrating to grpc avoids this and speeds the process up. Ill reword to make it more clear